mypy/test/teststubtest.py

```diff
@@ -929,6 +929,26 @@ class X(enum.Enum):
             """,
             error="X.c",
         )
+        yield Case(
+            stub="""
+            import enum
+            class Z(enum.IntEnum):
+                foo: int
+                @property
+                def spam(self) -> str: ...
```

Comment:
Do we need a failing test case for `spam: str` as a regular annotation? 🤔

Reply:
Good spot, the proposed new test fails :)